.ruby-gemset

```diff
@@ -0,0 +1,3 @@
+#Gems needed with version
```

Comment:
This is not how you create gemsets. Look at this - https://github.com/jf/rbenv-gemset
Have just the name e.g. `q_it` which is the gemset for the q_it project with Ruby version 2.5.0

Reply:
gotcha! Thanks for the example.